.travis.yml

```diff
@@ -111,3 +124,18 @@ script:
     else
       ctest -V
     fi
+  - |
+    if [[ "${DOCUMENTATION}" == "true" ]]; then
+      cd ..
+      doxygen doxy/doxygen.config
+      # Suppress output to avoid leaking the token when the command fails
+      git clone https://arximboldi:${GH_TOKEN}@github.com/ableton/atria --depth 1 --branch=gh-pages gh-pages &>/dev/null
+      rm -rf gh-pages/{search,*.png,*.css,*.js,*.html}
```

Comment:
To remove files generated previously that are no longer needed. Only top dir is populated by doxygen.